test/tests/fuel-cycle/tests

```diff
@@ -0,0 +1,10 @@
+[Tests]
+  issues = '#82'
+  design = 'ParsedODEKernel.md ODETimeDerivative.md'
+  [Abdou2021]
+    type = Exodiff
```

Comment:
I think this should be a csv diff test rather than an exodus file. (And I know @cticenhour and @ke7kto had a chat about using exodus, but I'll share arguments for csv files). 

Using exodus files for test is great when there are spatial variations in an output. Otherwise, I think a csv file is much, much better in this case for the following reasons:
- csv files are much easier to just open and plot (proof is, the doc plots were made using the csv output, not the exodus one). The reason why this is so important is because when the exodus file starts to diff, it is much more challenging to quickly identify exactly what makes it diff, whereas a quick plot of the csv files makes it obvious where the diff is. 
- csv files often uses less memory (granted these are small anyway, but still)

As someone who had to work with a lot of exodus tests that could have been csv files, I feel quite strongly about this. But as always, if we feel like exodus is necessary here, I'm happy to discuss it.

Reply:
Not necessary IMO, and I have had issues trying to read 1D exodus files using meshio in python, so I'm totally on board.

Reply:
I am fine with CSV for 0D cases like this. My main desire in the original discussion that I had with @ke7kto was to make sure we had output testing and not just RunApp testing.